output/components.sasslint.txt

```diff
@@ -1,11 +1,14 @@
 > sass-lint -v -q
 
 
+src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.scss
+  1:2  error  Class '.dropdownWrapper' should be written in lowercase with hyphens  class-name-format
```

Comment:
Linter error

Reply:
Can the linters (eslint and sasslint) checks be in requirements for the PR to be merged ?
It would avoid reviewers to check that, and prevent omissions :)

Reply:
done